apollo-integration/src/test/java/com/apollographql/apollo/ApolloCancelCallTest.java

```diff
@@ -13,23 +13,24 @@
 import org.junit.Test;
 
 import java.io.IOException;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 
+import okhttp3.Dispatcher;
 import okhttp3.OkHttpClient;
 import okhttp3.mockwebserver.MockResponse;
 import okhttp3.mockwebserver.MockWebServer;
 
 public class ApolloCancelCallTest {
-  private static final long TIME_OUT_SECONDS = 3;
   private ApolloClient apolloClient;
   @Rule public final MockWebServer server = new MockWebServer();
   private MockHttpCacheStore cacheStore;
 
   @Before
   public void setup() {
     cacheStore = new MockHttpCacheStore();
-    OkHttpClient okHttpClient = new OkHttpClient.Builder().build();
+    OkHttpClient okHttpClient = new OkHttpClient.Builder()
+        .dispatcher(new Dispatcher(Utils.immediateExecutorService()))
+        .build();
     apolloClient = ApolloClient.builder()
```

Comment:
We should set `.dispatcher(Utils.immediateExecutorService())` for `ApolloClient` as well, otherwise it will use the default one: `com.apollographql.apollo.ApolloClient.Builder#defaultDispatcher`

Reply:
Absolutely, good caught :)